pkg/daemon/gateway_linux.go

```diff
@@ -579,6 +591,13 @@ func (c *Controller) setIptables() error {
 					natPreroutingRules = append(natPreroutingRules, rule)
 					continue
 				case OvnPostrouting:
+					if util.ContainsString(rule.Rule, "MASQUERADE") && supportRandomFully {
+						//https://github.com/kubeovn/kube-ovn/issues/2641
```

Comment:
A space is needed between `//` and the comments

Reply:
fixed